src/Core/Prefab/GlobeView.js

```diff
@@ -198,7 +198,8 @@ function GlobeView(viewerDiv, coordCarto, options) {
     this.atmosphere.traverse((obj) => { obj.layers.set(atmosphereLayer); });
     this.camera.camera3D.layers.enable(atmosphereLayer);
 
-    this.scene.add(this.atmosphere);
+    wgs84TileLayer.object.add(this.atmosphere);
```

Comment:
I don't add them to the scene, their added to the `object` where the `tiles` of `wgs84TileLayer` will be added.